Transfer.go

```diff
@@ -91,7 +100,7 @@ func SendPingTx(param SendPingTxParam) (string, PingResultError) {
 					MicroLamports: param.ComputeUnitPrice,
 				}),
 				memoprog.BuildMemo(memoprog.BuildMemoParam{
-					Memo: []byte("ping"),
+					Memo: memoText,
```

Comment:
not sure how tight the current CU estimate is, but utf8-validating a longer string will surely take more